frontend/packages/console-app/src/components/network-policies/network-policy-form.tsx

```diff
@@ -215,14 +244,16 @@ export const NetworkPolicyForm: React.FunctionComponent<NetworkPolicyFormProps>
               name="denyAllIngress"
             />
           </div>
-          <div className="co-create-networkpolicy__deny-checkbox">
-            <Checkbox
-              label={t('public~Deny all egress traffic')}
-              onChange={handleDenyAllEgress}
-              checked={networkPolicy.egress.denyAll}
-              name="denyAllEgress"
-            />
-          </div>
+          {networkFeaturesLoaded && networkFeatures.PolicyEgress !== false && (
```

Comment:
It looks like these option will be hidden for users who don't have authority to request the network operator config? Is that the behavior we want?

Reply:
If the user don't have authority to request the network operation config, this option will be actually visible (`networkFeatures.PolicyEgress === undefined , which !== false`), and the console shows a warning message explaining that this feature is not available for OpenshiftsSDN.

For a second iteration, we are working in exposing the features as configMap that can be queried by any user.

Reply:
The explanation of the new configMap, as well as the usage of `true`, `false` and `undefined` for each element of the features map, are exposed here: https://github.com/openshift/enhancements/pull/875

Reply:
OK, I was assuming `networkFeaturesLoaded` would be `false` if the user didn't have authority, but that gets set to `true`?

Reply:
Yes. The `useClusterNetworkFeatures` function relies on the `useK8sGet` function and tries to be consistent with its behavior.

On error, `useK8sGet` function also returns `loaded == true`: https://github.com/openshift/console/blob/master/frontend/public/components/utils/k8s-get-hook.ts#L24-L26